share/completions/apt.fish

```diff
@@ -60,15 +60,24 @@ __fish_apt_option list -l all-versions -d 'Show all versions of any package'
 # Search
 __fish_apt_subcommand search -r -d 'Search for packages'
 
-# Search
+# Show
 __fish_apt_subcommand show -r -d 'Show package information'
 
+# Showsrc
+__fish_apt_subcommand showsrc -r -d 'Show information of source of given package'
```

Comment:
What does this description mean? What is the "information of source"?

Is that about the source package, or about the sources of the package?

Reply:
How about "Show source package information"?

The term "source package" is a thing in debian.